### ISSUE TITLE
Super Targeting

### ISSUE THREAD

**Issue by Mistarz91** [instabot-py/instabot.py#1134]
> Fill in all requirements *before* submitting an issue.
> Do not delete any of the template, fill all of it in; even if you think it doesn't apply to your issue.
> If you fail to follow these simple instructions, we will close the ticket.

- [x] This is a checked box. **Do not leave spaces around the `x`!**

- [x] I have searched for other issues with the same problem.

## Hosting
- [ ] Are you hosting the bot on a VPS or other Cloud hosting?
- [ ] If so; have you tried running the code on your local machine?

#### Select one:
- [ ] Bug
- [x] Feature Request
- [ ] Technical Help
- [ ] Other

### Operating System? (include version)
MAC / Windows / Linux / Android

## Python Version?
2 / 3

## Description

Type the description of your issue here.
I have looked for other feature requests for what you would call "Super Targeting", but they were all closed with a message to open a new issue if not solved.

So is it possible to get a feature (maybe it's already there and i just missed it) where you can put a id in for a page and then the bot target that pages followers and like a picture at every follower of that page for example. So you basically are trying to take the attention of users that you already know are liking a page like your own.
```
Include the contents of the error.log here (if applicable).
```

**Comment by instabot-master** [comment]
like that idea - plus a feature for direct messaging would be great.

**Comment by ChipWolf** [comment]
Yeah, I like the idea. PRs will be accepted for this.

**Comment by ittyGitty** [comment]
Sorry, might be intruding here, I just couldn't figure out where else to put my question.
Also I like the idea above! :)

I..! was wondering: 
Would it be possible to make the bot go from a post that is liked through to that owners feed, and like several of the photos there, before returning to the search of other posts to like from the tags provided?
(Haa, that was badly explained) The reason is that it gives more an impression of a real person if it seems that you have actually been to the profile and checked it out by liking several of the photos from the feed, not just one random.

Also, I would not* like to have any of my acquaintances to find my profile, haha (don't ask why), and it is a hassle to get all those usernames for the user_blacklist.
Maybe one could set some kind of location-manager to blacklist a whole.. country? 😇

Otherwise I love it <3

**Comment by squaredvision** [comment]
Yeah, I've been looking into find something like this, but I haven't seen anything out there yet! It'd be great to see this feature added.... A real game changer if it goes in.

**Comment by nosecretz** [comment]
any updates with this?
